webapp/src/pages/EditChatroomMemberPage/EditChatroomMemberPage.tsx

```diff
@@ -1,86 +1,207 @@
 import {
   AvatarPageTemplate,
+  ButtonProps,
   ButtonVariant,
   IconVariant,
-  Text,
-  TextColor,
-  TextVariant,
-  TextWeight,
+  Row,
+  ToggleSwitch,
 } from '../../shared/components';
+import './EditChatroomMemberPage.css';
+import {
+  AVATAR_EP_URL,
+  CHATROOM_EP_URL,
+  CHATROOM_URL,
+  USER_URL,
+  WILDCARD_AVATAR_URL,
+} from '../../shared/urls';
+import { useParams } from 'react-router-dom';
+import { useCallback, useEffect, useState } from 'react';
+import { chatApi, usersApi } from '../../shared/services/ApiService';
+import { useData } from '../../shared/hooks/UseData';
+import { Chatroom } from '../../shared/generated/models/Chatroom';
+import { UpdateChatroomMemberDto } from '../../shared/generated/models/UpdateChatroomMemberDto';
+import { useNavigation } from '../../shared/hooks/UseNavigation';
+import { ResponseError } from '../../shared/generated';
+import { useNotificationContext } from '../../shared/context/NotificationContext';
 import { useAuth } from '../../shared/hooks/UseAuth';
-import { AVATAR_EP_URL } from '../../shared/urls';
 
 export default function EditChatroomMemberPage() {
-  const { authUser, logout, isLoading: isAuthUserLoading } = useAuth();
-  return (
-    <AvatarPageTemplate
-      isLoading={isAuthUserLoading}
-      headerStatusVariant="online"
-      title={authUser?.username ?? ''}
-      avatarProps={{
-        url: `${AVATAR_EP_URL}/${authUser?.avatarId}`,
-        caption: 'level 4',
-        XCoordinate: authUser?.avatarX,
-        YCoordinate: authUser?.avatarY,
-      }}
-      button={{
+  const { chatroomId, username } = useParams();
```

Comment:
suggestions on how to split this page in different files are welcome. Anyway, we can also live with no splitting.

Reply:
Splitted.